user/advanced/alter.md

```diff
@@ -12,15 +12,15 @@ Alter an existing z/OS userid.
 ## `UserAdmin.alter()`
 
 ```python
-def alter(self, userid: str, traits: dict = {}) -> Union[dict, bytes]:
+def alter(self, userid: str, traits: dict) -> Union[dict, bytes]:
 ```
 
 #### 📄 Description
 
 &nbsp;
 
 {: .experimental }
-> _Only a subset of available **Segments** and **Traits** are considered **Stable**. See [Segments](../segments_traits_operators#segmetns) and [Traits](../segments_traits_operators#traits) for more details._
+> _Only a subset of available **Segments** and **Traits** are considered **Stable**. See [Segments](../advanced/segments_traits_operators#segmetns) and [Traits](../advanced/segments_traits_operators#traits) for more details._

```

Comment:
broken link.

Reply:
Change `#segmetns` to `#segments` in the url to segments documentation.

Reply:
that typo was in 3 places. All fixed